packages/application-shell/src/components/project-switcher/project-switcher.js

```diff
@@ -22,55 +22,60 @@ export const ValueContainer = props => (
   </div>
 );
 
-ValueContainer.propTypes = {
+ProjectSwitcherValueContainer.displayName = 'ProjectSwitcherValueContainer';
+ProjectSwitcherValueContainer.propTypes = {
   children: PropTypes.node.isRequired,
   projectCount: PropTypes.number.isRequired,
 };
 
-ValueContainer.displayName = 'ValueContainer';
-
-export const Option = props => (
+export const ProjectSwitcherOption = props => (
   <SelectInput.Option {...props}>
     <div className={styles['option-container']}>
       <div
         className={classnames(styles['item-text-main'], {
-          [styles['item-text-disabled']]:
-            (props.data.suspension && props.data.suspension.isActive) ||
-            (props.data.expiry && props.data.expiry.isActive),
+          [styles['item-text-disabled']]: props.isDisabled,
```

Comment:
The `isDisabled` prop is injected by react-select, based on the `isOptionDisabled` prop